modules/power-monitoring-uninstalling-pmo.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+
+// * power_monitoring/uninstalling-power-monitoring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="uninstalling-power-monitoring-operator_{context}"]
+= Uninstalling the {PM-operator} in the web console
+
+If you installed the {PM-operator} by using OperatorHub, you can uninstall it in the {OCP} web console.
+
+[WARNING]
+====
+Ensure that you have deleted the {PM-kepler} instance before uninstalling the {PM-operator}.
+====
+
+.Prerequisites
+* You have access to the {OCP} web console.
```

Comment:
Yes, that is exactly the case! I was so used to using {OCP} instead of {product-title} in the previous team that I did not realize we have it as {product-title} in the common attributes. :grinning:  So thank you for noticing and reminding me! :tada: